application/job/job.go

```diff
@@ -27,10 +27,10 @@ var handlers = map[string]func(worker.Args) error{
 	TokenCleanup:     tokenCleanupHandler,
 }
 
-func init() {
-	w = worker.NewSimple()
+func Init(appWorker *worker.Worker) {
+	w = appWorker
```

Comment:
Could you do this instead and keep w as worker.Worker? Would keep it simple later on, but I could see it maybe throwing errors too.
```
	w = *appWorker
```

Reply:
That's a good idea. I'll give it a shot.

Reply:
It worked in my local environment, but it wasn't enough to put the broken test back in. And I'm not sure about using a copy of the worker. Maybe best to keep it as a pointer to the original.